src/cpp/session/modules/SessionCodeTools.R

```diff
@@ -180,6 +180,109 @@
    !is.null(attr(func, "srcref"))
 })
 
+.rs.addFunction("parseSrcref", function(srcref)
+{
+   srcref <- as.list(unclass(srcref))
+   
+   # From ?srcref:
+   #
+   # Bytes (elements 2, 4) and columns (elements 5, 6) may be different due to
+   # multibyte characters. If only four values are given, the columns and bytes
+   # are assumed to match. Lines (elements 1, 3) and parsed lines (elements 7,
+   # 8) may differ if a #line directive is used in code: the former will respect
+   # the directive, the latter will just count lines. If only 4 or 6 elements
+   # are given, the parsed lines will be assumed to match the lines.
```

Comment:
Maybe it's just me, but I found this documentation to be nigh-on indecipherable...